code/game/gamemodes/antag_spawner.dm

```diff
@@ -119,16 +119,21 @@
 	target.equip_to_slot_or_del(new /obj/item/weapon/teleportation_scroll/apprentice(target), slot_r_store)
 
 /obj/item/weapon/antag_spawner/borg_tele
-	name = "Syndicate Cyborg Teleporter"
-	desc = "A single-use teleporter used to deploy a Syndicate Cyborg on the field."
+	name = "syndicate cyborg teleporter"
+	desc = "A single-use teleporter used to deploy a Syndicate cyborg on the field."
 	icon = 'icons/obj/device.dmi'
 	icon_state = "locator"
 	var/TC_cost = 0
+	var/borg_to_spawn
+	var/list/possible_types = list("assault", "medical")

```

Comment:
mind making this an associative list? it's slightly cleaner than using a switch()